src/metrics/repo_participants.rs

```diff
@@ -0,0 +1,133 @@
+use std::collections::HashMap;
+
+use anyhow::Error;
+use async_trait::async_trait;
+use chrono::{Duration, Utc};
+use fehler::throws;
+use graphql_client::GraphQLQuery;
+use tokio::sync::mpsc::Sender;
+
+use super::{Producer, GQL};
+
+pub struct RepoParticipants {
+    org_name: String,
+    repo_name: String,
+    number_of_days: i64,
+}
+
+impl RepoParticipants {
+    pub fn new(org_name: String, repo_name: String, number_of_days: i64) -> Self {
+        Self {
+            org_name,
+            repo_name,
+            number_of_days,
+        }
+    }
+}
+
+#[async_trait]
+impl Producer for RepoParticipants {
+    fn column_names(&self) -> Vec<String> {
+        vec![String::from("Participant"), String::from("PRs")]
+    }
+
+    async fn producer_task(self, tx: Sender<Vec<String>>) -> Result<(), anyhow::Error> {
+        let data = pr_participants(
+            &self.org_name,
+            &self.repo_name,
+            Duration::days(self.number_of_days),
+        )
+        .await
+        .unwrap();
```

Comment:
what's the significance of using `unwrap` instead of `?` here?

is this due to the fact that `pr_participants` does not return an error and thus you cannot propagate an error, so we panic if `await` fails?

Reply:
Actually, I think that's a holdover from the days when the error type was `String`, and I was too lazy to deal with it.

Reply:
haha fair enough, thanks for changing that to `anyhow::Error`… idk why that wasn’t my first choice. the `String` errors are far from elegant 😆